core/src/main/java/com/bankbazaar/tripmanager/repository/UsersRepository.java

```diff
@@ -4,7 +4,9 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 
+import java.util.Optional;
+
 public interface UsersRepository extends JpaRepository<Users,Long> {
     @Query("SELECT user FROM Users user WHERE user.email = ?1")
```

Comment:
Use the repository naming convention for these queries. No need for native queries. Check  [guide](https://docs.spring.io/spring-data/data-jpa/docs/2.5.x/reference/html/#jpa.query-methods.query-creation)

Reply:
I have changed user to u. as per what i have seen in the guide.